README.md

```diff
@@ -0,0 +1,52 @@
+# metricbeat
+
+Setup a metricbeat instance.
+
+
+## Requirements
+
+An apt based linux system
+
+## Role Variables
+
+
+| Variable           | Default / Mandatory                                                                                                                                  | Description                                                                                                                                                                                                                                              |
+|--------------------|------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `metricbeat_config`  | `{ output.elasticsearch: { hosts: ["localhost:9200"] }, metricbeat.config.modules: { reload.enabled: false } }` | Since the metricbeat settings file is a yaml file we write the whole object to the settings file. For more Information please see the [metricbeat settings file documentation](https://www.elastic.co/guide/en/metricbeat/current/metricbeat-settings-file.html) |
+| `metricbeat_modules` | `[]`                                                                                                                                                 | List of config files to write to modules.d/. More information below.                                                                                                                                                                                     |
+
+### `metricbeat_modules`
+Each entry in the `metricbeat_modules` consists out of the following entries.
+| Variable | Default / Mandatory | Description                                                                              |
+|----------|---------------------|------------------------------------------------------------------------------------------|
+| `name`   | :heavy_check_mark:  | Name of the module config file. This value has to be the same as the name of the module. |
+| `config` | :heavy_check_mark:  | Yaml config for the module name .                                                        |
+
+## Example Playbook
+
+```yml
+- hosts: all
+  become: true
+  vars:
+    metricbeat_config:
+      metricbeat.config.modules:
+        reload.enabled: false
+      output.logstash:
+        hosts: ["localhost:9200"]
+	metricbeat_modules:
```

Comment:
What's with the whitespace? See github web view.

Reply:
fixed

Reply:
unfortunately not